challenges/objects-arrays.js

```diff
@@ -90,32 +145,50 @@ const zooAnimals = [
 The zoos want to display both the scientific name and the animal name in front of the habitats. Populate the displayNames array with only the animal_name and scientific_name of each animal. displayNames will be an array of strings, and each string should follow this pattern: "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const displayNames = [];
-console.log(displayNames);
+
+//.forEach method works like a for loop 
+
+const displayNames = zooAnimals.forEach(beast => 
```

Comment:
have you done all the steps here for the question? what is actually happening to displayNames?

Reply:
I did not push().  I could try...
const displayNames = [];

zooAnimals.forEach(beast =>
  displayNames.push(
    `Name: ${beast.animal_name}, Scientific: ${beast.scientific_name}.`
  )
);
console.log(displayNames);

Reply:
const displayNames = [];

zooAnimals.forEach(beast => {
  displayNames.push(
    `Name: ${beast.animal_name}, Scientific: ${beast.scientific_name}.`
    );
})
console.log(displayNames);